Client/Frontend/Menu/ItemPages/MenuPageCollectionViewCell.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MenuPageCollectionViewCell.swift
+//  Client
+//
+//  Created by Emily Toop on 3/23/16.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
+
+import UIKit
+
+class MenuPageCollectionViewCell: UICollectionViewCell {
+
+    var pageView: MenuPageView = MenuPageView()
+
+    override init(frame: CGRect) {
+        super.init(frame: CGRectZero)
```

Comment:
Nit: You can just pass in the frame thats passed in above. Not sure what happens if you just set this to CGRectZero.